app/models.py

```diff
@@ -2873,12 +2873,24 @@ class ServiceJoinRequest(db.Model):
     def serialize(self) -> SerializedServiceJoinRequest:
```

Comment:
I like the User type on both Requester and Status Changed By
However, I think we should remove the requester_id from the model if we are using requester:User type

This means we would need refactoring on the table schema and on the `dao_create_service_join_request`. 
On `dao_create_service_join_request` we can leave the signature the same (requester_id) but change the logic of the function to get the User.

I just think that having both requester and requester_id in the class `ServiceJoinRequest` seems redundant.

Reply:
ignore the above, my brain clearly was not functioning properly  - sorry about that!